package.json

```diff
@@ -62,6 +62,7 @@
     "total.js": "^3.2.4",
     "trek-engine": "^1.0.5",
     "trek-router": "^1.2.0",
+    "uWebSockets.js": "github:uNetworking/uWebSockets.js#v15.11.0",
```

Comment:
could you target the [npm package](https://github.com/uNetworking/uWebSockets.js?files=1#package-install-anywhere)?

Reply:
Author did not publish this package at npm. As we know from a documentation, it is a right way to specify uws version.
https://github.com/uNetworking/uWebSockets.js

Reply:
Or you want see the 18.3.0 version here?

Reply:
Maybe the last version is better, but in the docs I read in docs that you can install with npm,but in npm there is not as you said.

Reply:
There was a dispute between the author of this package and npm. Ever since the package is not distributed via npm.
BTW I think it's better to update the package to the latest version.